### ISSUE TITLE
Python implementation for this project

### ISSUE THREAD

**Issue by woshipzs** [riejohnson/ConText#4]
Very nice paper. I am wondering if there is a python implementation for deep pyramid CNN models. I am new to running deep learning on C++. Thanks!

**Comment by riejohnson** [comment]
Hi.  Thanks for your interest in our work.  Sorry, but there is no plan for a Python implementation or a Python wrapper.  In the past, people have attempted to port it to tensorflow or pyTorch, but if I remember it correctly, there was some difficulty in handling variable-sized documents as variable-sized (which is what this code does) without padding or truncating in those platforms.
